node/test/@opent2t/test-b/TranslatorBC.js

```diff
@@ -1,78 +0,0 @@
-"use strict";
-// This code uses ES2015 syntax that requires at least Node.js v4.
-// For Node.js ES2015 support details, reference http://node.green/
-
-const EventEmitter = require("events");
-
-/**
- * This translator class implements two interfaces: "org.opent2t.test.B"
- * and "org.opent2t.test.C". Since those interfaces do not have any conflicting
- * member names, they are both implemented directly.
- */
-class TestTranslatorBC extends EventEmitter {
-
-    constructor(deviceProps) {
```

Comment:
I think I recall we decided to stick with initDevice pattern for now. That's what I have in my PR, at least.